crates/findex/Cargo.toml

```
@@ -72,49 +73,46 @@ backend-wasm = [
 
 [dependencies]
 async-trait = { workspace = true }
-# Optional dependencies
-# This is a `backend-redis` test dependency that conflicts with `backend-wasm`
-# WASM test compilation. Passing it as an optional dependency is dirty I
-# couldn't come up with another fix for now.
-cosmian_crypto_core = { workspace = true, default-features = false, features = ["ser", "sha3"]}
+cosmian_crypto_core = { workspace = true, default-features = false, features = [
+  "ser",
+  "sha3",
+] }
 cosmian_findex = { git = "https://github.com/Cosmian/findex", branch = "fix/cloudproof_rust_integration" }
 
 # Optional dependencies
-actix-rt = { version = "2.8", optional = true}
+actix-rt = { version = "2.9", optional = true }
 base64 = { workspace = true, optional = true }
 cosmian_ffi_utils = { workspace = true, optional = true }
-hex = { workspace = true, optional = true }
 js-sys = { workspace = true, optional = true }
-futures = { version = "0.3.28", optional = true }
+futures = { version = "0.3.29", optional = true }
 log = { version = "0.4.20", optional = true }
-pyo3 = { workspace = true, features = ["extension-module"], optional = true }
+pyo3 = { workspace = true, optional = true}
 rand = { workspace = true, optional = true }
-redis = { version = "0.23", default-features = false, features = ["connection-manager", "script", "tokio-comp"], optional = true }
-reqwest = { version = "0.11.20", default-features = false, optional = true }
+redis = { version = "0.23", features = [
+  "aio",
+  "ahash",
+  "script",
+  "connection-manager",
+  "tokio-comp",
+], optional = true }
+reqwest = { version = "0.11.22", features = [
+  "rustls-tls",
+], default-features = false, optional = true }
```

Do we really need TLS here? All Findex communication is encrypted by design.
```suggestion
reqwest = { version = "0.11.22", default-features = false, optional = true }
```

---

feature removed